pkg/render/policyrecommendation.go

```diff
@@ -139,6 +141,25 @@ func (pr *policyRecommendationComponent) clusterRole() client.Object {
 			},
 			Verbs: []string{"create", "delete", "get", "list", "patch", "update", "watch"},
 		},
+		{
+			// Add write access to Linseed APIs.
+			APIGroups: []string{"linseed.tigera.io"},
+			Resources: []string{"events"},
+			Verbs:     []string{"create"},
+		},
+		{
+			// Add read access to Linseed APIs.
+			APIGroups: []string{"linseed.tigera.io"},
+			Resources: []string{
+				"waflogs",
+				"dnslogs",
+				"l7logs",
+				"flowlogs",
```

Comment:
We probably only need flowlogs from this list.

Reply:
Do we need both flows and flowlogs ?

Reply:
I've removed `flowlogs`